src/Components/People.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react'
+import axios from "axios"
+
+export default class People extends Component {
+    constructor() {
+        super()
+        this.state = {
+            searchCharacter: "",
+            currentMovie: {},
```

Comment:
this is a pretty misleading choice of name!!! Maybe `currentCharacter` would be more appropriate?